inspirehep/dojson/common/base.py

```diff
@@ -318,3 +322,70 @@ def deleted_records2marc(self, key, value):
     return {
         'a': inspire_dojson_utils.get_recid_from_ref(value)
     }
+
+
+@conferences.over('urls', '^8564')
+@experiments.over('urls', '^8564')
+@hep.over('urls', '^856.[10_28]')
+@hepnames.over('urls', '^856.[10_28]')
+@institutions.over('urls', '^856.[10_28]')
+@jobs.over('urls', '^856.[10_28]')
+@journals.over('urls', '^856.[10_28]')
+def urls(self, key, value):
+    """URL to external resource."""
+    urls = []
+    value = utils.force_list(value)
+    for val in value:
+        url_dict = {}
+        description = utils.force_list(val.get('y'))
+        url_values = utils.force_list(val.get('u'))
+        for u in url_values:
```

Comment:
Again, to make sure, shouldn't the `descriptions` and `url_values` be zipped in this traversal? Seems that every url will get the same description. Is this correct?